ci-operator/step-registry/mirror-images/by-oc-adm/mirror-images-by-oc-adm-commands.sh

```diff
@@ -20,15 +20,21 @@ if [ ! -f "${SHARED_DIR}/mirror_registry_url" ]; then
 fi
 echo "MIRROR_REGISTRY_HOST: $MIRROR_REGISTRY_HOST"
 
+if [[ -n "$OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE" ]] && [[ X"$RELEASE_IMAGE_LATEST" != X"$OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE" ]]; then
```

Comment:
The `[[ X"$RELEASE_IMAGE_LATEST" != X"$OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE" ]]` is a redundant check, which is not needed.
You can check with "OVERRIDE=aaa, LATEST=bbb", and "OVERRIDE=ccc, LATEST=ccc"

Reply:
thanks, in this case, if OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE is non null, then will use OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE as source,  so removing [[ X"$RELEASE_IMAGE_LATEST" != X"$OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE" ]]

Reply:
Hello, 

We were discussing of adding that further check (the inequality) to stay safe and because the first one (non empty string) seems always satisfied: e.g., the install step will fail if not https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh#L248-L252

@liangxia if you are sure they are always the same, shall we remove the if at all and only use `"$OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE"`?